handlers/metrics.go

```diff
@@ -7,7 +7,7 @@ import (
 var (
 	RedirectHandlerRedirectCountMetric = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
-			Name: "router_redirect_handler_redirect_count",
+			Name: "router_redirect_handler_redirect_total",
```

Comment:
Is there a way to introduce `promtool` into the automated tests? I'd probably forget to run `curl http://localhost:8081/metrics | promtool check metrics` manually.

Reply:
In order to do this, `promtool` must be available in Jenkins which is a separate PR

Until then I've added a slightly (very) hacky make task to do it in de41b23